encrypt/encrypt.go

```diff
@@ -14,11 +23,11 @@ USAGE: %s encrypt [file(s)]
 
```

Comment:
Seems that line 22 should be 
 `USAGE: %s encrypt -key <public-key-file> [file(s)]`

Reply:
Yes! definitely!